source/camera/advanced/capture_hdr_print_normals.py

```diff
@@ -5,7 +5,6 @@
 
 """
 
-
```

Comment:
intentionally removed @andersfagerli ?

Reply:
Yes, but should fix this in SDK instead. Currently waiting on the doctag fixes before doing anything here.